### DISCUSSION TITLE
How many seconds will it lose the reward?

### DISCUSSION THREAD

**Discussion by qianmyth** [Chia-Network/chia-blockchain#6382] [Support]
If the harvester accesses the file for more than 5 seconds, there will be a risk. How many seconds will it lose the reward?

**Comment by cccat6** [comment]
30s, including the network and various factors.

**Comment by BrandtH22** [comment]
@qianmyth , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
